src/main/java/domain/piece/piecerole/Pawn.java

```diff
@@ -0,0 +1,68 @@
+package domain.piece.piecerole;
+
+import domain.move.Direction;
+import domain.move.Movable;
+import domain.piece.Color;
+import domain.position.Position;
+import java.util.List;
+import java.util.Objects;
+
+public class Pawn implements PieceRole {
+    private List<Movable> routes;
+    private int moveCount;
+
+    public Pawn(final Color color) {
+        moveCount = 1;
+        if (color == Color.BLACK) {
+            this.routes = List.of(
+                    new Movable(getMaxMovement(moveCount), Direction.SOUTH),
```

Comment:
`getMaxMovement()` 메서드는 이 곳에서 초기화 할 때 밖에 사용되지 않고 있어요. 
한번 움직임을 수행한 후에는 값을 직접 감소시키고요. 

아래와 같이 매직넘버를 사용해 주거나, 해당 값을 상수로 추출하는 것은 어떤가요?
```suggestion
                    new Movable(2 , Direction.SOUTH),
```

Reply:
아하, 다시 보니 `SOUTH` 방향에만 2를 사용하고, 나머지 방향에는 1을 사용해 주기 위함이었던 것 같네요. 
하지만, 동일하게 메서드 대신 값을 직접 명시해 주는 것이 훨씬 가독성에 좋을 것 같습니다.